express-session/express-session.d.ts

```diff
@@ -10,10 +10,12 @@ declare namespace Express {
 
   export interface Request {
     session?: Session;
+    sessionId?: string;
```

Comment:
Is this property name `sessionID`? via. https://github.com/expressjs/session#reqsessionid-1